src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js

```diff
@@ -271,7 +271,7 @@
                  * @param order
                  */
                 insertOrderParamsInUrl = function(url, order) {
-                    if (!!order && !!order.length) {
```

Comment:
seems that they was saved as an array before. but now its only an object like: `{"attribute":"lastName","direction":"asc"}`